### PR TITLE
Continued updates to NFS documentation

### PR DESCRIPTION
**Pull request by bmcelvee**
Continuing edits from: https://github.com/openshift/openshift-docs/pull/11947

@sferich888

### CODE REVIEW

File: scaling_performance/optimizing_storage.adoc [hunk 1]
```diff
@@ -101,6 +101,15 @@ The following table summarizes the recommended and configurable storage technolo
...
 
+[IMPORTANT]
+====
+For metrics it is an anti-pattern to use any shared storage and a single volume
+(RWX). By default, metrics deploys with one volume per Cassandra Replica.  
+
+For logging, one volume per loggin-es is required. Therefore using any shared
+storage would be an anti-pattern.
+====
+
```
Comment (sferich888) [inline scaling_performance/optimizing_storage.adoc]: Can we connect this to the tabel in some way (foot note)? 

I wonder if some astrics for the file row is ndded for rwx and logging and metrics that points to this (call-out).

Comment (bmcelvee) [reply scaling_performance/optimizing_storage.adoc]: Yeah, I added the statements as footnotes to logging and metrics. For the asterisks for RWX that point to the logging and metrics footnotes, I'm going to need to give it a bit more thought on how to execute that.

Thanks!

**Comment by bmcelvee** [comment]
Thanks again, @sferich888! I updated all of the warnings to be specific to the sections they're referencing.

@openshift/team-documentation PTAL :maple_leaf:

File: scaling_performance/optimizing_storage.adoc [hunk 2]
```diff
@@ -87,8 +87,10 @@ The following table summarizes the recommended and configurable storage technolo
 | Yes
 | Configurable
 | Configurable
-| Configurable
-| Configurable
+a| Configurable footnoteref:[metrics-warning,For metrics it is an anti-pattern to use any shared storage and a single volume
+(RWX). By default, metrics deploys with one volume per Cassandra Replica.]
+a| Configurable footnoteref:[logging-warning,For logging, one volume per logging-es is required. Therefore using any shared
+storage would be an anti-pattern.]
```
Comment (mburke5678) [inline scaling_performance/optimizing_storage.adoc]: @bmcelvee Do you need the `a` before the row? The footnote below does not use the `a`. 
Also, it seems a bit awkward that the first footnote starts with the anti-pattern statement then the OpenShift configuration; but, the second is reversed. 
s/For metrics/For metrics, (add a comma for consistency between the two.

**Comment by mburke5678** [comment]
@bmcelvee A few nitty things. Otherwise LGTM.

File: scaling_performance/optimizing_storage.adoc (hunk 2 above)
Comment (bmcelvee) [reply scaling_performance/optimizing_storage.adoc]: Thanks, @mburke5678! I reordered the statements to make them line up better. For the`a`, I had several ci-test failures and put it in as an effort to resolve them. All fixed!

**Comment by bmcelvee** [comment]
Manually CP'ing 3.7-3.9 since there will be conflicts.

**Pull request merged**